recipe/meta.yaml

```diff
@@ -11,8 +11,8 @@ source:
 
 build:
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
-  skip: True  # [win or py==38]
+  number: 2
+  skip: True  # [py==38]
```

Comment:
This selector can likely be removed entirely as conda-forge is python3.9+.  Are you happy with the CI result?